imports/lib/utils.js

```diff
@@ -86,10 +88,21 @@ export function hideModalAlert () {
   }, 600)
 }
 
+export function resetAlert () {
+  Session.set('globalAlertShow', null)
+  Session.set('globalAlertMessage', null)
+  Session.set('modalAlertShow', null)
+  Session.set('modalAlertMessage', null)
+}
+
 export function removeTrailingSlash (str) {
   return str.replace(/\/$/, '')
 }
 
+export function __ (message) {
```

Comment:
I think we should rename this function to be called `translateText` or something. Even if `__` is consistent with how we will translate text in templates, this naming is too cryptic in my opinion.

Reply:
@bent0b0x yes you are right, if we use just **text** or **transText** ?

Reply:
Sorry, I disagree. The underscore is an accepted convention for marking strings to be translated, and it is good (I think, in this case, by way of exception) to have a very unobtrusive function call for marking strings, because it will be all over the place while not really changin any of the logic

Reply:
I'd actually prefer the single underscore, just as in the templates :-)

Reply:
@jellegerbrandy fair enough! 👍